_includes/head.html

```diff
@@ -20,31 +20,42 @@
     
     {% assign pagetitle = page.title | strip_newlines %}
 
-    <meta name="twitter:card" content="summary" />
-    <meta name="twitter:title" content="{% if pagetitle != "" %}{{ page.title }}{% else %}OpenSearch{% endif %}" />
-    {% if page.twittercard.account %}
-    <meta name="twitter:site" content="{{ page.twittercard.account }}" />
-    {% elsif page.authors %}
-      {% assign author_short_name = page.authors | first  %}
-      {% assign author = site.authors | where: 'short_name', author_short_name | first %}
-      {% if author.twitter %}
-        <meta name="twitter:site" content="@{{ author.twitter }}" />
-      {% endif %} 
+    {% if page.twittercard.image or page.feature_image %}
+    <meta name="twitter:card" content="summary_large_image">
+    {% else %}
+    <meta name="twitter:card" content="summary">
     {% endif %}
 
```

Comment:
I believe that we can also specify `player` for the meta keyword `twitter:card` - we may want to some day tweet something that plays automagically for people on twitter.

Reply:
@nateynateynate Would a reasonable solution be to accept a specific card type indicated in the Front Matter and then fall back accordingly if not specified?

Reply:
@nateynateynate I have updated the `head.html` template with the player type logic.